worker-queue-rabbit/src/main/java/com/github/workerframework/queues/rabbit/TransientDeliveryException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2025 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.workerframework.queues.rabbit;
+
+public class TransientDeliveryException extends Exception {
```

Comment:
Does this need to be public?

```suggestion
class TransientDeliveryException extends Exception {
```